packages/flutter_tools/analysis_options.yaml

```diff
@@ -3,6 +3,10 @@
 
 include: ../../analysis_options.yaml
 
+analyzer:
```

Comment:
Should this be in //dev/conductor instead?

Reply:
oops, yeah, lol

Reply:
thanks, I was so frustrated as to why this was still failing